src/Sylius/Component/Core/Inventory/Operator/ORMOrderInventoryOperator.php

```diff
@@ -0,0 +1,101 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Paweł Jędrzejewski
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Sylius\Component\Core\Inventory\Operator;
+
+use Doctrine\DBAL\LockMode;
+use Doctrine\ORM\EntityManagerInterface;
+use Doctrine\ORM\OptimisticLockException;
+use Sylius\Component\Core\Model\OrderInterface;
+use Sylius\Component\Core\Model\OrderItemInterface;
+
+/**
+ * @author Grzegorz Sadowski <grzegorz.sadowski@lakion.com>
+ */
+final class ORMOrderInventoryOperator implements OrderInventoryOperatorInterface
```

Comment:
Maybe we should move the file to `Sylius\Component\Core\Doctrine\ORM` (similar to bundles) or to (`Sylius\Bundle\CoreBundle\Doctrine\ORM`)?